loops-and-arrays.js

```diff
@@ -36,6 +46,11 @@ const numbers = [2, 4, 5, 29, 38];
 // ==========================================
 
 const squares = [30, 2, 8, 24, 11];
+console.log(squares)
+
+for(let i = 0; i < squares.length; i++)
```

Comment:
ik mis de opening en closing tag van je forloop { }, zelfde als bij de vorige

hij logt hem wel goed, dat is dan wel weer grappig. hierbij moet de console.log ook weer buiten de forloop worden geplaatst